.github/CODEOWNERS

```
@@ -0,0 +1,71 @@
+# Centrifudge Code Owners
+
+# Changes to .github
+.github/* @mikiquantum @mustermeiszer @branan @NunoAlexandre @offerijns
+
+# Changes to ci
+ci/* @mikiquantum @mustermeiszer @branan @NunoAlexandre @offerijns
+
+# Changes to the service of our chain.
+src/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
+
+# Changes to chain-specs
+res/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
+
+# Changes to specific pallets
+## Connectors related logic
+pallets/connectors/* @NunoAlexandre
```

I think I can be a backup for this.
```suggestion
pallets/connectors/* @NunoAlexandre @offerijns
```